internal/policy/verify.go

```diff
@@ -643,3 +649,46 @@ func getChangedPaths(repo *git.Repository, entry *rsl.ReferenceEntry) ([]string,
 
 	return gitinterface.GetDiffFilePaths(currentCommit, priorCommit)
 }
+
+func checkCommitAgainstModifiedPaths(ctx context.Context, repo *git.Repository, commit *object.Commit, commitPolicy *State) ([]string, error) {
+	filePaths, err := gitinterface.GetFilePathsChangedByCommit(repo, commit)
+	if err != nil {
+		return []string{}, err
+	}
+
+	badPaths := []string{}
+	for _, path := range filePaths {
+		keys, err := commitPolicy.FindPublicKeysForPath(ctx, fmt.Sprintf("file:%s", path))
```

Comment:
bump on this.

Reply:
I also want us to start thinking about this in the context of attestations, because a policy might require more than 1 approval for a file change...

Reply:
I have changed it to use FindVerifiersForPath, instead of FindPublicKeysForPath, and the verification is currently using the current policies TargetEnvelope. When PR #261 lands, this will have to change, and the whole delegation tree will have to be traversed

Reply:
I have changed this to use the current attestations in the repo.